### DISCUSSION TITLE
I cannot login to Vercel

### DISCUSSION THREAD

**Discussion by LnAHTS** [vercel/vercel#4399] [Help] [answered]
I login with my email (I'm not a dev) and since Vercel added a step where I need to receive an email to login, and it never arrives (yes, I have checked my spam folder) and tried several times. 

I need to get into my account. Does anyone have a brilliant workaround? 

I have emailed support, but I don't know how long it will take for them to respond. 

Thanks

**Comment by paulogdm** [comment, marked as answer]
My apologies for the delay. I'm closing this discussion in favor of your ticket.

Account-related issues are better handled in our support channel, for security concerns.
